docs/app/pages/charts/charts-sections/timeline-chart/timeline-chart/timeline-chart.component.ts

```diff
@@ -133,6 +155,9 @@ export class ChartsTimelineChartComponent extends BaseDocumentationSection imple
         modules: [{
             library: 'chart.js'
         },
+        {
+            library: 'moment'
+        },
```

Comment:
I tried adding this here to make the plunker for the timeline chart work however I haven't had much sucess. Any help with this would be appreciated. Other than that, this is ready for review